woocommerce/woocommerce
# Fix visual inconsisties in Cart/Checkout between elements and between Light/Dark modes

This PR is a massive style fixing PR, it goes over several sections in Cart/Checkout and fixes broken styles, mostly around dark mode, but also to make things consistent.

- Shipping select (Ship vs Pickup), Method Selector, Payment Selector, Address Card, Inputs, Select, and Checkboxes all now have consistent select, focus, and default modes.
- All outlines are now 1.5px
- All colors in dark mode are consistent.
- Textarea is now consistent with rest of items.
- Outlines in Cart/Checkout are now visible in dark mode.
- Input labels are centered.
- Address Change button in cart is now correctly styled.



Closes https://github.com/woocommerce/woocommerce/issues/49706
Closes https://github.com/woocommerce/woocommerce/issues/49476
Closes https://github.com/woocommerce/woocommerce/issues/48945
Closes https://github.com/woocommerce/woocommerce/issues/48842
Closes https://github.com/woocommerce/woocommerce/issues/48816
Closes https://github.com/woocommerce/woocommerce/issues/48620
Closes https://github.com/woocommerce/woocommerce/issues/45179
Closes https://github.com/woocommerce/woocommerce/issues/42054



### How to test the changes in this Pull Request:

This is a visual PR, to fully test it, you should pick 2 themes (Storefront and a default theme like 2024) and go over Cart/Checkout elements.

1. Make sure Local Pickup is enabled.
2. Go over Cart/Checkout in storefront, make sure no styling look broken, out of place, and inconsistent.
3. Focus mainly on focus states, hover.
4. Switch to 2024 and do the same testing.
5. In 2024, switch to a dark color scheme.
6. Go to editor, and change Cart/Checkout to use dark mode colors.
7. Go over the same flow again, making sure everything is visible and colors are consistent.



## Screenshots
| Before | After |
|--------|-------|
| <img width="469" alt="Screenshot 2024-09-20 at 4 16 49 PM" src="https://github.com/user-attachments/assets/47393e14-60ec-42a2-b1bc-5d42125f4ad5"> | <img width="462" alt="Screenshot 2024-09-20 at 4 15 47 PM" src="https://github.com/user-attachments/assets/4b316202-68b7-40dc-b441-2256781410cb"> |
| <img width="801" alt="Screenshot 2024-09-20 at 4 12 26 PM" src="https://github.com/user-attachments/assets/7dc82e8f-475b-4ced-bdef-0b4d5e320f20"> |    <img width="803" alt="Screenshot 2024-09-20 at 4 15 03 PM" src="https://github.com/user-attachments/assets/2a2d3b0f-854e-44fb-a318-5c871194004f"> |
| <img width="245" alt="Screenshot 2024-09-20 at 4 03 59 PM" src="https://github.com/user-attachments/assets/98b9ec8b-d939-4ba6-b33b-2bae85c361a2"> |     <img width="300" alt="Screenshot 2024-09-20 at 4 05 55 PM" src="https://github.com/user-attachments/assets/fd7b21df-19f4-4737-acd6-3a92da73f69c"> |
| <img width="101" alt="Screenshot 2024-09-20 at 4 03 52 PM" src="https://github.com/user-attachments/assets/3968416e-eba2-4d8e-ac8c-806529e09eed"> |    <img width="144" alt="Screenshot 2024-09-20 at 4 19 51 PM" src="https://github.com/user-attachments/assets/8f7a91a1-eed2-4dfb-b168-53b366ff8247"> |
| <img width="750" alt="Screenshot 2024-09-20 at 3 56 58 PM" src="https://github.com/user-attachments/assets/fcfb6faa-dbcc-41c8-b01d-3ae144540108"> |   <img width="651" alt="Screenshot 2024-09-20 at 3 57 02 PM" src="https://github.com/user-attachments/assets/d314ce2f-5c56-4466-bf50-eadc7cdf70ae"> |
| <img width="420" alt="Screenshot 2024-09-20 at 3 58 35 PM" src="https://github.com/user-attachments/assets/d4e9efee-f31a-4e55-9f92-3798285c8e58"> |   <img width="652" alt="Screenshot 2024-09-20 at 3 57 06 PM" src="https://github.com/user-attachments/assets/5c4a58e1-8a0c-4c8a-a065-fea96da6ab7e"> |
| <img width="420" alt="Screenshot 2024-09-20 at 3 58 35 PM" src="https://github.com/user-attachments/assets/e40143c3-2240-4c5b-a5a5-40e6da96623b"> |   <img width="648" alt="Screenshot 2024-09-20 at 3 57 26 PM" src="https://github.com/user-attachments/assets/d60ef531-25b3-47c4-919d-ffc68e8b4490"> |
| <img width="227" alt="Screenshot 2024-09-20 at 3 58 43 PM" src="https://github.com/user-attachments/assets/eb018b62-c77e-4f09-9b7a-a645c5b3b1da"> |    <img width="301" alt="Screenshot 2024-09-20 at 3 59 10 PM" src="https://github.com/user-attachments/assets/e29f41ca-54be-4d8f-aca4-dcf27ccfc96f"> |
| <img width="449" alt="Screenshot 2024-09-20 at 3 59 20 PM" src="https://github.com/user-attachments/assets/a41ee2e4-1be3-4f44-a2b0-de4a7060466f"> |     <img width="561" alt="Screenshot 2024-09-20 at 3 59 36 PM" src="https://github.com/user-attachments/assets/f5846c7e-4bad-40b3-8a65-79a661b1133f"> |
| <img width="412" alt="Screenshot 2024-09-20 at 3 59 54 PM" src="https://github.com/user-attachments/assets/bde20082-7cf5-4a75-a926-e15350f74b19"> |   <img width="554" alt="Screenshot 2024-09-20 at 4 00 32 PM" src="https://github.com/user-attachments/assets/992c28b1-1c0d-4a07-ae94-ba7199c1a3c7"> |
| <img width="218" alt="Screenshot 2024-09-20 at 4 00 52 PM" src="https://github.com/user-attachments/assets/7ae683d7-d6d3-44c3-8881-ba4176543db5"> |    <img width="202" alt="Screenshot 2024-09-20 at 4 00 46 PM" src="https://github.com/user-attachments/assets/8a6c01a0-f25d-4f3e-b7ec-ea7d778688ac"> |
| <img width="422" alt="Screenshot 2024-09-20 at 4 02 25 PM" src="https://github.com/user-attachments/assets/3a9a2d1e-c978-41b4-9510-66c026ab23ae"> |   <img width="579" alt="Screenshot 2024-09-20 at 4 02 14 PM" src="https://github.com/user-attachments/assets/90f970db-bf04-4aa5-b09c-f0487e2f938e"> |
| <img width="431" alt="Screenshot 2024-09-20 at 4 02 33 PM" src="https://github.com/user-attachments/assets/f5ae6cf6-6767-4c78-841e-c9acf498bb89"> |    <img width="584" alt="Screenshot 2024-09-20 at 4 02 09 PM" src="https://github.com/user-attachments/assets/6382b2fa-47b0-4287-a31e-ee97cc0f09c9"> |
| <img width="688" alt="Screenshot 2024-09-20 at 4 03 48 PM" src="https://github.com/user-attachments/assets/54a19c52-5f5a-4fe7-9d61-28f991a2e015"> |    <img width="907" alt="Screenshot 2024-09-20 at 4 05 42 PM" src="https://github.com/user-attachments/assets/a67bffec-7ffd-44a7-9ad2-d15a75927201"> |

### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

Improve visual styles for Cart/Checkout in light and dark modes.

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
